docs/sources/installation/debian.md

```diff
@@ -37,22 +37,22 @@ Install any missing dependencies:
 sudo apt-get install -y software-properties-common wget
 ```
 
```

Comment:
FYI, I tested the installation process you are rewriting on a total of five versions of Ubuntu and Debian. It works as written. Have you tested this new process?